packages/user/src/migrations/Migration20240213092656.ts

```diff
@@ -1,22 +1,43 @@
-import { Migration } from '@mikro-orm/migrations';
+import { Migration } from "@mikro-orm/migrations"
```

Comment:
comment,very-nit(this-file): Think we should be consistent in casing, but we can clean up later on